aries_cloudagent/revocation/models/issuer_rev_reg_record.py

```diff
@@ -305,14 +305,7 @@ async def send_def(
 
         return rev_reg_res
 
-    async def send_entry(
-        self,
-        profile: Profile,
-        options: Optional[dict] = None,
-        write_ledger: bool = True,  # TODO Delete me
-        endorser_did: str = None,  # TODO Delete me
-    ) -> RevListResult:
-        """Send a registry entry to the ledger."""
+    def ready_to_send_check(self):
```

Comment:
In the mindset of wanting to eliminate usage of this class if we can, I think most of these checks are "internal" checks about the integrity of the record as opposed to something that must strictly be done before updating a revocation list. I think we can omit this.